cmd/piped/BUILD.bazel

```diff
@@ -18,9 +19,52 @@ go_binary(
     visibility = ["//visibility:public"],
 )
 
+# Create a passwd file with a root and pipecd user and uid.
```

Comment:
We have to load the needed functions before using them.

```
load(
    "@io_bazel_rules_docker//contrib:passwd.bzl",
    "passwd_entry",
    "passwd_file",
)

load(
    "@io_bazel_rules_docker//container:container.bzl",
    "container_image",
)
```

Reply:
Damm... I still don't get the dependency structures in Bazel.